lib/hammer_cli_foreman_remote_execution/job_invocation.rb

```diff
@@ -51,8 +53,25 @@ class InfoCommand < HammerCLIForeman::InfoCommand
         end
       end
 
+      def adapter
+        :yaml
+      end
+
       def extend_data(invocation)
-        JobInvocation.extend_data(invocation)
+        if option_show_inputs?
+          invocation['template_invocations']&.each do |template|
+            input_values = template['template_invocation_input_values']
+            values_for_host = {}
+            input_values&.each do |input_value|
+              values_for_host[input_value['template_input_name']] = input_value['value']
+            end
+            hosts = invocation.dig('targeting', 'hosts')
+            host_index = hosts&.index { |h| h['id'] == template['host_id'] } || template['host_id']
```

Comment:
I'm afraid you can't use host id as a fallback value instead of the index

Reply:
why not?

Reply:
Index will be a value in the range from 0 to NUMBER_OF_HOSTS-1. Host ids can be arbitrary, well outside of this range. For example you could end up using number 7000 to index an array of 100 elements